azure-pipelines.dfe.yml

```diff
@@ -103,7 +103,7 @@ jobs:
 
   - job: 'Admin'
     pool:
-      vmImage: 'ubuntu-16.04'
```

Comment:
Is there no 20.04 image? May as well use the latest LTS version if its available.

Reply:
There is a 20.04 image - probably should use that: https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml

Unless there is some reason I'm unaware of?